test/utilities/DominoDocumentFragment.test.js

```diff
@@ -0,0 +1,27 @@
+import assert from 'assert'
+import domino from 'domino'
+import dominoDocumentFragment from './DominoDocumentFragment'
+
+const documentFragmentFromHTMLString = dominoDocumentFragment.documentFragmentFromHTMLString
+
+describe('DominoDocumentFragment', () => {
+  describe('documentFragmentFromHTMLString()', () => {
+    it('should return a Domino DocumentFragment when given an HTML string', () => {
+      const frag =
+        documentFragmentFromHTMLString('<div>text</div><a id="link1"></a>')
+      assert.ok(frag instanceof domino.impl.DocumentFragment)
+    })
```

Comment:
Done! I wonder if we can update linting to enforce our preference here... too easy to miss 'em.